net/matrix-commander/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                matrix-commander
+python.rootname     matrix_commander
+version             8.0.4
+revision            0
+categories-append   net chat
+supported_archs     noarch
+platforms           {darwin any}
+license             LGPL-3
+maintainers         nomaintainer
+description         Simple command-line Matrix client
+long_description    {*}${description}.
+homepage            https://github.com/8go/matrix-commander
+checksums           rmd160  8f008284ec54b4c34a6c7bbebaae29e4bcf7cb72 \
+                    sha256  464f7a4e3a632f5f13f831b882484d91552e3868292a63b9671cc1562f513fef \
+                    size    198269
+supported_archs     noarch
```

Comment:
no need to duplicate the line )here and in other ports). It 's preferred up next to the "platforms" line